code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -102,7 +113,7 @@
 	anchored = 0
 	density = 0
 	flavour_text = "<font size=3><b>Y</b></font><b>ou are a Free Golem. Your family worships <span class='danger'>The Liberator</span>. In his infinite and divine wisdom, he set your clan free to \
-	travel the stars with a single declaration: \"Yeah go do whatever.\" Though you are bound to the one who created you, it is customary in your society to repeat those same words to newborn \
+	travel the stars with a single declaration: \"Fucking ghost roles just want free antag\" Though you are bound to the one who created you, it is customary in your society to repeat those same words to newborn \
```

Comment:
Oh yes, certainly. Golems are the best, close to my heart, one of my first changes was the golem naming system.